crates/matrix-sdk-ui/src/timeline/builder.rs

```diff
@@ -172,7 +172,7 @@ impl TimelineBuilder {
         let (room_event_cache, event_cache_drop) = room.event_cache().await?;
         let (_, event_subscriber) = room_event_cache.subscribe().await;
 
-        let is_live = matches!(focus, TimelineFocus::Live { .. });
+        let is_live = matches!(focus, TimelineFocus::Live { .. } | TimelineFocus::Thread { .. });
```

Comment:
Since `is_live` is used downstream, how about making the downstream users take some variant of the focus? (either the simplified focus kind, or the full focus as a reference) so as to make the consequence of being live closer to the decision? Even `is_pinned_events` below would likely benefit from such a treatment, IMO